base/applications/osk/lang/pt-PT.rc

```diff
@@ -0,0 +1,84 @@
+/*
+ * PROJECT:         ReactOS Kernel
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * FILE:            base/applications/osk/lang/en-US.rc
+ * PURPOSE:         On screen keyboard (Portuguese resource).
+ * PROGRAMMERS:     Copyright Denis ROBERT
+ * TRANSLATION:     Valmor Flores (valmorflores@gmail.com)
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_DEFAULT
```

Comment:
LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL

Reply:
Thank you. I was in doubt, as there were files with both forms.